### ISSUE TITLE
BibTeX to Biber suggestion

### ISSUE THREAD

**Issue by Sparklester** [Pseudomanifold/latex-mimosis#12]
Why not change the template's BibLaTeX backend to Biber? 

It seems like Biber is more reliable when paired with BibLaTeX, and its also being actively developed.

I switched the template's setting to Biber a few days ago and it has been working great so far.

**Comment by Pseudomanifold** [comment]
That's a good point! When I started this template, support for Biber was more limited. But you are using the Biber configuration on Overleaf even? I think I will make it possible to configure this behaviour and select Biber as the default. What do you think about that?

**Comment by Sparklester** [comment]
Yes, the Biber configuration works perfectly by just changing it from BibTeX to Biber in BibLaTeX's options.

Both Overleaf with their custom TeX Live 2019 version and TeXmaker (with MiKTeX and a Biber configuration) have been able to compile the project properly at the first go. From there I could also quickly make the citations APA 7th with just a few changes in the bibliography-mimosis.tex file.

Making Biber the default seems like a safe bet in my opinion.

**Comment by Pseudomanifold** [comment]
Thanks, fixed it now!
